src/main/scala/BintrayPlugin.scala

```diff
@@ -0,0 +1,194 @@
+package bintray
+
+import bintry.Attr
+import sbt.{ AutoPlugin, Credentials, Global, Path, Resolver, Setting, Task, Tags, ThisBuild }
+import sbt.Def.{ Initialize, setting, task, taskDyn }
+import sbt.Keys._
+import sbt.Path.richFile
+
+object BintrayPlugin extends AutoPlugin {
+  import BintrayKeys._
+  import InternalBintrayKeys._
+
+  override def requires = sbt.plugins.JvmPlugin
+  override def trigger = allRequirements
```

Comment:
A frustrating thing about this is for builds where you have some things that you want published to bintray (eg sbt plugins) and some things that you don't (eg everything else to maven central), you need to explicitly disable the bintray plugin for all the projects that don't want it.